tests/php/UtilsTest.php

```diff
@@ -938,6 +936,20 @@ public function test_set_media() {
 		Utils\set_media( $post_id, [ $media_item ], [ 'use_filesystem' => false ] );
 	}
 
+	/**
+	 * Data provider for test_set_media
+	 *
+	 * @since 1.0
+	 * @group Utils
+	 */
+	public function data_test_set_media() {
```

Comment:
Nit: I recommend removing the `test_`, that way we have a test prefix for tests and a data prefix for data providers.

I also don't have super strong feeling so ignore if you have your own preferences. This is the first data provider so we can use this to set a convention.

Reply:
Updated in https://github.com/10up/distributor/pull/1296/commits/264bd4b0e881722cb941fe39c30a709025d07c78. Forgot to remove `test_` after the copy function name. Thanks for the note.